week1/homework/src/server.js

```diff
@@ -2,19 +2,27 @@
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
-function createServer(port) {
+const sendState = require('./responses/sendState');
+const sendError = require('./responses/sendError');
+
+const createServer = port => {
   let state = 10;
 
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+  return http.createServer(({ url }, response) => {
+    url === '/'
+      ? response.writeHead(302, { Location: '/state' })
```

Comment:
This is unreadable with ternarie operators... Please change this to a switch...

Reply:
Hello Josja. 
In my opinion more readable than switch, a bit more readable then if-else. 
No problem I'll fix it today.